### PR TITLE
dotCMS/core#14574 Changes to content on a page not reflected on re-edit

### PR DESCRIPTION
**Pull request by fmontes**

### CODE REVIEW

File: src/app/portlets/dot-edit-page/content/services/dot-edit-content-html/dot-edit-content-html.service.ts [hunk 1]
```diff
@@ -480,6 +466,10 @@ export class DotEditContentHtmlService {
         return contentletEventsMap[event];
     }
 
+    private shouldUpdateContentletInode(target: HTMLElement) {
+        return target.dataset.dotObject === 'edit-content' && target.tagName === 'BUTTON';
+    }
+
```
Comment (freddyucv) [inline src/app/portlets/dot-edit-page/content/services/dot-edit-content-html/dot-edit-content-html.service.ts]: why do we need this check?

File: src/app/portlets/dot-edit-page/content/services/dot-edit-content-html/dot-edit-content-html.service.ts [hunk 2]
```diff
@@ -192,17 +193,6 @@ export class DotEditContentHtmlService {
         this.currentAction = DotContentletAction.ADD;
     }
 
-    /**
-     * Set the container id where a contentlet will be update
-     *
-     * @param {string} identifier
-     * @memberof DotEditContentHtmlService
-     */
-    setContainterToEditContentlet(pageContainer: DotPageContainer): void {
```
Comment (freddyucv) [inline src/app/portlets/dot-edit-page/content/services/dot-edit-content-html/dot-edit-content-html.service.ts]: COOL!!!!

File: src/app/portlets/dot-edit-page/content/services/dot-edit-content-html/dot-edit-content-html.service.ts (hunk 1 above)
Comment (fmontes) [reply src/app/portlets/dot-edit-page/content/services/dot-edit-content-html/dot-edit-content-html.service.ts]: Becuase we only need to update the contentlet inode in the DOM when we're editing from the pencil edit button at the top, not if we edit the vtl or the shortcut edit in a listing like in news.

Comment (freddyucv) [reply src/app/portlets/dot-edit-page/content/services/dot-edit-content-html/dot-edit-content-html.service.ts]: but  the value of dotObject is 'contentlet' in that case, isn't it?

Comment (fmontes) [reply src/app/portlets/dot-edit-page/content/services/dot-edit-content-html/dot-edit-content-html.service.ts]: `target` will be always a button to edit.

**Pull request merged**